crates/sui-protocol-config/src/lib.rs

```diff
@@ -1099,6 +1106,7 @@ impl ProtocolConfig {
                 let mut cfg = Self::get_for_version_impl(version - 1);
                 cfg.feature_flags
                     .disallow_change_struct_type_params_on_upgrade = true;
+                cfg.feature_flags.no_extraneous_module_bytes = true;
```

Comment:
@tzakian has 8 gone out yet?

Reply:
> 

I think PV 8 is already cut.
Change to 9